tools/xtce_generation/telemetry_generator.py

```diff
@@ -189,16 +231,24 @@ def create_container_set(
         self, telemetry_metadata, ccsds_parameters, container_name
     ):
         """
-        Create XML elements for ContainerSet, CCSDSPacket SequenceContainer,
+        Create XML elements.
+
+        These elements are for ContainerSet, CCSDSPacket SequenceContainer,
         and Packet SequenceContainer.
 
-        Parameters:
-        - telemetry_metadata: The TelemetryMetaData element where containers are.
-        - ccsds_parameters: A list of dictionaries containing CCSDS parameter data.
-        - container_name: The name of sequence container
+        Parameters
+        ----------
+        telemetry_metadata : iterable TODO Double Check
+            The TelemetryMetaData element where containers are.
+        ccsds_parameters : iterable TODO double check
+            A list of dictionaries containing CCSDS parameter data.
+        container_name : iterable TODO double check
```

Comment:
I think container_name is a string. There's one other place to update lower down.